server/repositories/mongo/mongo-messages-repository.ts

```diff
@@ -0,0 +1,392 @@
+import {
+  Message,
+  MessageWithVotes,
+  VoteMessage
+} from '../../models/message.js';
+import { snapshotHubMongoDb } from '../../index.js';
+import { MessagesRepository } from '../messages-repository.js';
+import { toVotesMessageJson } from '../../helpers/utils.js';
+
+const getMessages: (
+  space: string,
+  msgType: string
+) => Promise<Message[]> = async (space, msgType) => {
+  const db = await snapshotHubMongoDb;
+  const messagesCollection = db.collection(
+    process.env.MONGODB_TRIBUTE_DAOS_SNAPSHOT_HUB_DB_MESSAGES_COLLECTION_NAME
+  );
+  return messagesCollection
+    .find<Message>({ space, type: msgType }, { sort: { timestamp: -1 } })
+    .toArray()
+    .then((result: Message[]) => {
+      console.log(result.length);
```

Comment:
Sorry about all these, not sure they serve any purpose. I think they're holdovers from the original author who we forked from ages ago, Fabian @ Snapshot Labs.